### PR TITLE
Basic structure

### PR DESCRIPTION
**Pull request by Lordkaito**
In this branch:
1. Set webpack to work with my project.
2. Added basic HTML and CSS that will be modified later dynamically.
3. Added basic js to add content to my page.

### CODE REVIEW

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Lordkaito, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The project has an array of simple to-do tasks.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Live Demo
+
+[Live Demo](https://silly-poincare-6174f7.netlify.app/)
+
+
+### Prerequisites
+
+Dont need any prerequisites to get started.
+
```
[the remark this answers is not in the document]
Comment (rloterh) [reply README.md]: This issue is well resolved! ✔️

File: src/style.css [hunk 2]
```diff
@@ -1,7 +1,77 @@
...
+  width: 100%;
+}
+
+.container {
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
+  max-width: 500px;
+  width: 100%;
+  height: 100%;
+  max-height: 300px;
+  display: flex;
```
[the remark this answers is not in the document]
Comment (rloterh) [reply src/style.css:33]: This issue is well resolved! ✔️

File: src/index.js [hunk 3]
```diff
@@ -1,20 +1,52 @@
-import _ from 'lodash';
+// import _ from 'lodash';
```
[the remark this answers is not in the document]
Comment (rloterh) [reply src/index.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Isaí, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer(s). There is nothing else to say other than... it's time to merge it 🍾:shipit:.
Congratulations! 🎉🥂


Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**